Makefile

```diff
@@ -26,12 +26,24 @@ all: $(PREFIX_PROG_STRIPPED)lwip
 LWIPDIR := lib-lwip/src
 include $(LWIPDIR)/Filelists.mk
 LWIP_EXCLUDE := netif/slipif.c
+
+ifeq (${LWIP_G3_BUILD}, yes)
```

Comment:
this is ok for now, I think we may want to change it in the future to either use the "driver" semantics similar to below or go for `lwipopts.h` altogether.

I assume we can't currently build `g3`/*` and `netif/lowpan6.c`? If so, I'm ok with excluding these files permanently for now (hopefully forever).

Not a merge blocker.